packages/core/content-type-builder/package.json

```diff
@@ -63,8 +63,8 @@
   "peerDependencies": {
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-router-dom": "5.3.4",
-    "styled-components": "5.3.3"
+    "react-router-dom": "^5.3.4",
```

Comment:
Not sure we need to change `react-router-dom` because it doesn't go higher than `5.3.4` for v5, wdyt?

Reply:
Sure that makes sense.